buildspec.pypi.yml

```diff
@@ -18,8 +19,10 @@ phases:
       - echo -e "\nInstalling dependencies"
       - pip install poetry
       - poetry install
+      - pip install dbt-platform-helper
 
   build:
     commands:
       - echo -e "\nBuild and publish to PyPi"
       - ./publish_to_pypi.sh ${PYPI_TOKEN} ${SLACK_TOKEN} ${SLACK_CHANNEL_ID}
+      
```

Comment:
github complaining about lack of newline at EOF, despite newline...

Reply:
@gabelton Have removed newline to follow pattern of other buildspec files